src/table/columnCheckboxes.component.tsx

```diff
@@ -1,29 +1,20 @@
 import React from 'react';
-import { Column } from 'react-table';
 import { Checkbox } from '@mui/material';
 import { useAppDispatch, useAppSelector } from '../state/hooks';
 import {
-  selectSelectedColumns,
+  selectSelectedChannels,
   selectColumn,
   deselectColumn,
 } from '../state/slices/tableSlice';
 import { useVirtualizer } from '@tanstack/react-virtual';
-import { useAvailableColumns } from '../api/channels';
+import { useChannels } from '../api/channels';
+import { FullChannelMetadata } from '../app.types';
 
 const ColumnCheckboxes = React.memo((): React.ReactElement => {
-  const { data: availableColumns } = useAvailableColumns();
+  const { data: channels } = useChannels();
 
-  const selectableColumns = React.useMemo(
-    () =>
-      availableColumns?.filter((col) => {
-        const accessor = col.accessor?.toString();
-        return accessor && accessor.toUpperCase() !== 'TIMESTAMP';
-      }) ?? [],
-    [availableColumns]
-  );
```

Comment:
If timestamp is now treated like a normal channel, it is returned by `useChannels` right? Which means we need code to filter out timestamp again

Reply:
You're absolutely right, another thing I must have missed. I'll also add a unit test to check for this in column checkboxes